pages/windows/set.md

```diff
@@ -0,0 +1,19 @@
+# set
+
+> Display or set environment variables for the current instance of CMD.
+
+- List all current environment variables:
+
+`set`
+
+- Set an environment variable to a specific value:
+
+`set {{name}}={{value}}`
+
+- List environment variables starting with the specified string:
+
+`set {{name}}`
+
+- Prompt the user for a value for the specified variable:
```

Comment:
What is the use for this ? Batch scripts ?

Reply:
Indeed! It's like `read -p` in bash.

Reply:
Sorry, looks like my comment didn't post as I was on the train.

Yes, it's mainly used for batch scripts.